backend/src/graphql/resolver/ContributionMessageResolver.test.ts

```diff
@@ -94,5 +264,84 @@ describe('ContributionMessageResolver', () => {
         )
       })
     })
+
+    describe('authenticated', () => {
+      beforeAll(async () => {
+        await userFactory(testEnv, bibiBloxberg)
+        await userFactory(testEnv, peterLustig)
+        await query({
+          query: login,
+          variables: { email: 'bibi@bloxberg.de', password: 'Aa12345_' },
+        })
+        result = await mutate({
+          mutation: createContribution,
+          variables: {
+            amount: 100.0,
+            memo: 'Test env contribution',
+            creationDate: new Date().toString(),
+          },
+        })
+        await query({
+          query: login,
+          variables: { email: 'peter@lustig.de', password: 'Aa12345_' },
+        })
+        await mutate({
+          mutation: adminCreateContributionMessage,
+          variables: {
+            contributionId: result.data.createContribution.id,
+            message: 'Admin Test',
+          },
+        })
+        await query({
+          query: login,
+          variables: { email: 'bibi@bloxberg.de', password: 'Aa12345_' },
+        })
+        await mutate({
+          mutation: createContributionMessage,
+          variables: {
+            contributionId: result.data.createContribution.id,
+            message: 'User Test',
+          },
+        })
+      })
+
+      afterAll(async () => {
+        await cleanDB()
+        resetToken()
+      })
+
+      it('returns a list of contributionmessages', async () => {
+        await expect(
+          mutate({
+            mutation: listContributionMessages,
+            variables: { contributionId: result.data.createContribution.id },
+          }),
+        ).resolves.toEqual(
+          expect.objectContaining({
+            data: {
+              listContributionMessages: {
+                count: 2,
+                messages: expect.arrayContaining([
+                  expect.objectContaining({
+                    id: expect.any(Number),
+                    message: 'Admin Test',
+                    type: 'DIALOG',
+                    userFirstName: 'Peter',
+                    userLastName: 'Lustig',
+                  }),
+                  expect.objectContaining({
+                    id: expect.any(Number),
+                    message: 'User Test',
+                    type: 'DIALOG',
+                    userFirstName: 'Bibi',
+                    userLastName: 'Bloxberg',
+                  }),
+                ]),
+              },
+            },
+          }),
+        )
+      })
+    })
```

Comment:
Hey!
Very good tests @Elweyn !!! 😍

Do we really need to clean the database after the tests?
And may be don't need the creations in the different `beforeAll`.
Or could build all up and query for the build list of messages at the end?

It is really nice like it is. Just to fulfill a master piece … ?!

And to save time in testing …